packages/ui/src/components/Modal/__stories__/WithTooltip.stories.tsx

```diff
@@ -0,0 +1,15 @@
+import type { StoryFn } from '@storybook/react'
+import { Button } from '../../Button'
+import { Tooltip } from '../../Tooltip'
+import { Modal } from '../index'
+
+export const WithTooltip: StoryFn = props => (
+  <Modal {...props} disclosure={<Button>With a Tooltip</Button>}>
+    <Tooltip
+      placement="top"
+      text="⚠️&nbsp;For now, this does not work, tooltip appear below the Modal!"
```

Comment:
Can you replace the text by something else like "the tooltip should be on top of the modal". Im afraid we will forget about this text